dataservice/api/participant/resources.py

```diff
@@ -27,8 +31,15 @@ def get(self):
             resource:
               Participant
         """
+        q = (Participant.query
+                        .options(joinedload(Participant.diagnoses))
+                        .options(joinedload(Participant.samples))
```

Comment:
are we writing similar lines in samples resources as well?

Reply:
Yeah, we'll want to do some basic benchmarks as we implement different resources to make sure we're doing things effectively.